Add a simple description to this policy (#25)
gravitee-io/gravitee-policy-keyless: README.adoc

== Description

> This security policy consider any request as valid and so will not block any.

Reviewer: ```suggestion
This security policy does not block any requests as it considers them as valid by default.
```

During its execution the policy will set multiple attributes:

> * `application`: Anonymous application default value i.e. `1`

Author: For this one the meaning is `application` is always set to the anonymous application value which is equal to 1

> * `application`: Anonymous application default value i.e. `1`

Reviewer: Sure, so something like this perhaps?

* `application`: Anonymous application value, which is equal to `1` by default.

> * `application`: Anonymous application default value i.e. `1`

Reviewer: Even better

> * `application`: Anonymous application default value i.e. `1`

Author: Thanks! I just removed the `by default` as it's not the default value but THE value
